### ISSUE TITLE
Release 2023.09.1

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/benefits#1735]
### Release manager

@thekaveman

### Release version

2023.09.1

### What type of release is this?

Regular

### Regular release checklist

- [ ] Ensure the `dev` branch and secrets are up to date
- [ ] Create a branch called release/version from the source branch
- [ ] Bump the application version
- [ ] Open a PR for the release branch into `dev`, merge
- [ ] Ensure `test` secrets are up to date
- [ ] Open another PR from `dev` to `test`, merge
- [ ] QA the app in test
- [ ] Ensure `prod` secrets are up to date
- [ ] Open a PR for the `test` branch into `prod`, merge
- [ ] QA the app in prod
- [ ] Tag the release on the `prod` branch, push the tag to GitHub (see [docs](https://docs.calitp.org/benefits/deployment/release/#5-tag-the-release) for commands)
- [ ] Create a release in GitHub for the tag, generating release notes
- [ ] Edit release notes with additional context, images, animations, etc. as-needed

### Hotfix release checklist

- [ ] Make the hotfix in a branch from `prod`
- [ ] Bump the application version
- [ ] Ensure `prod` secrets are up to date
- [ ] Open a PR from the hotfix branch into `prod`, merge
- [ ] QA the fix in prod
- [ ] Tag the release on `prod`, push the tag to GitHub (see [docs](https://docs.calitp.org/benefits/deployment/release/#5-tag-the-release) for commands)
- [ ] Create a release in GitHub for the tag, generating release notes
- [ ] Edit release notes with additional context, images, animations, etc. as-needed
- [ ] Create a branch off `dev`
- [ ] Open a PR from `prod` into that branch, merge
- [ ] (optional) Adapt hotfix to current state of `dev` in that same branch
- [ ] Merge the PR into `dev`

**Comment by thekaveman** [comment]
During QA we discovered an issue with a race condition between code changes and app settings changes, documented here: #1742

**Comment by angela-tran** [comment]
#1739 and #1741 were needed to get in the change from #1738

**Comment by thekaveman** [comment]
@indexing: the latest code and configuration are deployed to production.

We've assigned you to unblock this issue as soon as we are ready to proceed with making a release (e.g. turning the feature on, posting release notes).

**Comment by indexing** [comment]
- Functionality is available in prod
- Release notes are pubished: https://github.com/cal-itp/benefits/releases/tag/2023.09.1
